governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+**Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?**
+
+Yes, recent events over the EDSR made it very clear ambiguity exists. Various community members were unsure on if this clause would allow for such a vote. One reasonable interpretation is that this power was meant to be used to fix obvious errors or weird edge cases that break things, but the author clarified that it could be used to deal with broader protocol issues(such as low DSR utilization and declining DAI supply). 
```

Comment:
You haven't actually identified the ambiguities that are there though. You've just said they exist based on recent events.